assets/stylesheets/index.scss

```
@@ -1,6 +1,13 @@
 // Base
-@import 'base/header';
+@import 'base/form';
```

Do we need to have prefix for imported file?
```suggestion
@import 'base/_form';
```
And also the file name, should be `_form.scss`?